data-processing/data_manipulation/common/config.py

```diff
@@ -30,6 +31,11 @@ class Config(metaclass=Singleton):
     
     def __init__(self, yaml_file_path='config.yml'):
         logger.debug(f"{log_tag_const.CONFIG} start to load config file.")
+
+        file = Path(yaml_file_path)
+        if not file.exists():
+            yaml_file_path=os.path.join(os.getcwd(), 'data_manipulation/config.yml')
```

Comment:
for config.yaml, if we allow user to select model service, do we still need configuration like zhipuai，llm.use_type and open_ai?

Reply:
don't need